src/structures/Message.js

```diff
@@ -460,7 +462,10 @@ class Message extends Base {
    * }
    */
   async crosspost() {
-    await this.client.api.channels(this.channel.id).messages(this.id).crosspost.post();
+    await this.client.api
+      .channels(this.channel.id)
+      .messages(this.id)
+      .crosspost.post();
```

Comment:
Please revert those changes, it's causing lint to fail.

Reply:
I was having the same issue in my PR; looks like eslint inside VSCode automatically changes it to multiple lines and shows errors if it's one line, but the cli fails if it's one line.

Reply:
eslint automatically formatted this in WebStorm, wasn't a change I actively made 😂

Reply:
Oh wait, this happens in the pre-commit hooks...